src/seaport.ts

```diff
@@ -130,6 +131,185 @@ export class OpenSeaPort {
     this._emitter.removeAllListeners(event)
   }
 
+  /**
+   * Wraps an arbirary group of NFTs into their corresponding WrappedNFT ERC20 tokens.
+   * Emits the `WrapAssets` event when the transaction is prompted.
+   * @param param0 __namedParameters Object
+   * @param tokenIds Token IDs of the assets.
+   * @param tokenAddresses Addresses of the tokens' contracts. Must be the same length as `tokenIds`. Each address corresponds with its respective token ID in the `tokenIds` array.
+   * @param accountAddress Address of the user's wallet
+   */
+  public async wrapAssets(
+      { tokenIds, tokenAddresses, accountAddress }:
+      { tokenIds: string[];
+        tokenAddresses: string[];
+        accountAddress: string; }
+    ) {
+
+    if (!tokenIds || !tokenAddresses || tokenIds.length != tokenAddresses.length) {
+      throw new Error("The 'tokenIds' and 'tokenAddresses' arrays must exist and have the same length.")
+    }
+
+    // Check if all tokenAddresses match. If not, then we have a mixedBatch of
+    // NFTs from different NFT core contracts
+    const isMixedBatchOfAssets: boolean = !tokenAddresses.every( (val, i, arr) => val === arr[0] )
+
+    const assets: Asset[] = _.zipWith(tokenIds, tokenAddresses, (tokenId, tokenAddress) => {
+      return { tokenAddress, tokenId }
+    })
+
+    this._dispatch(EventType.WrapAssets, { assets, accountAddress })
+
+    const gasPrice = await this._computeGasPrice()
+    const txHash = await sendRawTransaction(this.web3, {
+      from: accountAddress,
+      to: WRAPPED_NFT_LIQUIDATION_PROXY_ADDRESS_MAINNET,
```

Comment:
Can we set `this.wrappedNFTLiquidationProxyAddress` in the constructor based on the current network name? e.g. `this.wrappedNFTLiquidationProxyAddress = this._networkName == Network.Main ? WRAPPED_NFT_LIQUIDATION_PROXY_ADDRESS_MAINNET : WRAPPED_NFT_LIQUIDATION_PROXY_ADDRESS_RINKEBY`